form.go

```diff
@@ -90,6 +93,9 @@ func NewForm(groups ...*Group) *Form {
 		teaOptions: []tea.ProgramOption{
 			tea.WithOutput(os.Stderr),
 		},
+
+		SubmitCmd: tea.Quit,
+		CancelCmd: tea.Quit,
```

Comment:
Let's keep these in `Run` since this was intentional to avoid having these pre-set in embedded forms in bubble tea applications.